ms/MSOper/MSMetaData.cc

```diff
@@ -3571,9 +3913,22 @@ void MSMetaData::_getScanAndSubScanProperties(
     }
     SHARED_PTR<std::map<SubScanKey, SubScanProperties> > myssprops;
     SHARED_PTR<std::map<ScanKey, ScanProperties> > myscanprops;
+#ifdef _OPENMP
+    if (_parallel) {
+        _computeScanAndSubScanPropertiesParallel(
+            myscanprops, myssprops, showProgress
+        );
+    }
+    else {
+        _computeScanAndSubScanProperties(
+            myscanprops, myssprops, showProgress
+        );
+    }
+#else
     _computeScanAndSubScanProperties(
         myscanprops, myssprops, showProgress
     );
+#endif
```

Comment:
For reasons I don't immediately understand, the original serial method takes about 8 minutes to run on my machine, whereas running the multi-threaded method in serial mode takes about 2.5 minutes. So I think I can safely discard the original method.